Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function priceWithVat(price){
+    let percentage = 1.2;
+    return `£${(price * percentage)}`; 
```

Comment:
Between number and string types, which type do you think is more appropriate for the returned value of this function? Why?